card/db/migrate_core_cards/20150220134731_following_to_follow_rule.rb

```diff
@@ -0,0 +1,19 @@
+# -*- encoding : utf-8 -*-
+
+class FollowingToFollowRule < Card::CoreMigration
+  def up
+    Card.search(:right=>{:codename=>'following'}, :left=>{:type=>'user'}).each do |following_card|
+      user_name = following_card.cardname.left
+      following_card.item_names.each do |followed_set_name|
+        
+        set_card = Card.fetch(follwed_set_name, :new=>{})
+        if (!set_card.type_code == :set)
+          set_card = set_card.default_follow_set_card
+        end
+        rule = Card.fetch set_card.follow_rule_name(user_name), :new=>{:type=>'pointer'}
+        rule.content = "[[always]]"
```

Comment:
I think you can put this in the :new params.  Can it exists already?  If so, then I guess this needs to be separate.  Unless it is a warning/error situation if it exists.  Then we should test whether it is new.